submissions/ksalpern/Hooli-style-Popup/scss/style.scss

```diff
@@ -0,0 +1,280 @@
+        align-items: flex-end;
+        text-align: center;
+        margin-bottom: 8px;
+        .popup__item-icon {
+          display: inline-block;
+          width: 70px;
+          height: fit-content;
+          text-align: center;
+          cursor: pointer;
+          border: 1px solid transparent;
+          border-radius: 4px;
+          -webkit-border-radius: 4px;
+          -moz-border-radius: 4px;
+          -ms-border-radius: 4px;
+          -o-border-radius: 4px;
+          margin-bottom: 8px;
+          padding: 4px 0;
+
+          .popup__item-name {
+            display: block;
+            color: rgb(27, 24, 24);
+            font-size: 12px;
+            padding-top: 12px;
+          }
+          .popup__item-name:hover {
+            text-decoration: underline;
+          }
+        }
+        .popup__item-icon:hover {
+          background-color: #d1cbcb;
+        }
+        .popup__item-icon:focus {
+          background-color: #aaa5a5;
+          outline: none;
+        }
+      }
+    }
+  }
+}
+
+.popup__content::-webkit-scrollbar {
+  width: 4px;
+}
+.popup__content::-webkit-scrollbar-thumb {
+  border-radius: 4px;
+  -webkit-border-radius: 4px;
+  -moz-border-radius: 4px;
+  -ms-border-radius: 4px;
+  -o-border-radius: 4px;
+  background-color: #d1cbcb;
+}
+.popup__content::-webkit-scrollbar-track {
+  margin: 4px;
+}
+
+.icon {
+  max-height: 36px;
+}
+
+.popup__more {
+  .more {
+  }
+  .popup__more-items {
+    display: none;
+    flex-wrap: wrap;
+    margin-top: 12px;
+    .popup__more-item {
+      .popup__more-icon {
+        display: inline-block;
+        width: 80px;
+        height: fit-content;
+        text-align: center;
+        cursor: pointer;
+        border: 1px solid transparent;
+        border-radius: 4px;
+        -webkit-border-radius: 4px;
+        -moz-border-radius: 4px;
+        -ms-border-radius: 4px;
+        -o-border-radius: 4px;
+        margin-bottom: 8px;
+        padding: 4px 0;
+
+        .popup__more-name:hover {
+          text-decoration: underline;
+        }
+      }
+      .popup__more-icon:hover {
+        background-color: #d1cbcb;
+      }
+      .popup__more-icon:focus {
+        background-color: #aaa5a5;
+        outline: none;
+      }
+    }
+  }
+}
+
+.even-more {
+  width: 100%;
+}
+.even-more:hover {
+  text-decoration: underline;
+}
+
+.nav__item_bell {
+  border-radius: 4px;
+  -webkit-border-radius: 4px;
+  -moz-border-radius: 4px;
+  -ms-border-radius: 4px;
+  -o-border-radius: 4px;
+  padding: 4px;
+  .bell {
+    width: 20px;
+  }
+}
+.nav__item_bell:focus,
+.nav__item_bell:hover,
+.nav__item_bell:focus-visible {
+  background-color: rgb(109, 221, 221);
+  outline: none;
+}
+
+.user-item {
+  border-radius: 50%;
+  -webkit-border-radius: 50%;
+  -moz-border-radius: 50%;
+  -ms-border-radius: 50%;
+  -o-border-radius: 50%;
+  padding: 4px;
+  .user {
+    background-color: rgb(180, 235, 226);
+    color: rgb(56, 53, 49);
+    img {
```

Comment:
Avoid styling by HTML tag. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)